libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/MyTurnContext.java

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.ai.qna;
+
+import com.microsoft.bot.builder.BotAdapter;
+import com.microsoft.bot.builder.DeleteActivityHandler;
+import com.microsoft.bot.builder.SendActivitiesHandler;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.builder.TurnContextStateCollection;
+import com.microsoft.bot.builder.UpdateActivityHandler;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.ConversationReference;
+import com.microsoft.bot.schema.InputHints;
+import com.microsoft.bot.schema.ResourceResponse;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class MyTurnContext implements TurnContext {
+
+    private BotAdapter adapter;
+    private Activity activity;
+
+    public MyTurnContext(BotAdapter withAdapter, Activity withActivity) {
+        this.adapter = withAdapter;
+        this.activity = withActivity;
+    }
+
+    public String getLocale() {
+        throw new UnsupportedOperationException();
+    }
+
+    public void setLocale(String withLocale) {throw new UnsupportedOperationException();}
```

Comment:
Why some are one-liners and some are multi lines methods? I assume it is because of the length limit, but in that case I would prefer consistency.

Reply:
You're right here!, I'll fix that